zigbee2mqtt/readme.rst

```diff
@@ -0,0 +1,134 @@
+===========
+zigbee2mqtt
+===========
+
+Das Plugin dienst zur Steuerung von Zigbee Devices via Zigbee2MQTT über MQTT. Notwendige Voraussetzung ist eine
+funktionsfähige und laufende Installation von Zigbee2Mqtt. Die Installation, Konfiguration und der Betrieb ist hier
+beschrieben: https://www.zigbee2mqtt.io/
+Dort findet man ebenfalls die unterstützten Zigbee Geräte.
+
+.. attention::
+
+    Das Plugin kommuniziert über MQTT und benötigt das mqtt Modul, welches die Kommunikation mit dem MQTT Broker
+    durchführt. Dieses Modul muß geladen und konfiguriert sein, damit das Plugin funktioniert.
+
+Getestet ist das Plugin mit folgenden Zigbee-Geräten:
+
+- SONOFF SNZB-02
+- IKEA TRADFRI E1766
+- Aqara DJT11LM
+- TuYa TS0210
+- Aqara Opple 3fach Taster
+
+
+Konfiguration
+=============
+
+Die Informationen zur Konfiguration des Plugins sind in der **plugin.yaml** nachzulesen.
```

Comment:
Das ist unglücklich. Ein Anwender sollte in die plugin.yaml eines Plugins eigentlich nie rein schauen (müssen). Dafür wird die Konfigurationsdoku ja generiert und ist direkt in der Anwender Doku zugänglich. Falls Du von hier auf diese Seite verweisen willst, müsstet Du hier eine Referenz zu der generierten Seite einfügen.

Beim hue2 Plugin ist das z.B. so realisiert. Dort steht in der user_doc.rst
```
Diese Parameter und die Informationen zur Item-spezifischen
Konfiguration des Plugins sind unter :doc:`/plugins_doc/config/hue2` beschrieben.
```

für dieses Plugin müsstest Du also ``:doc:`/plugins_doc/config/zigbee2mqtt` `` schreiben.

Reply:
Das arbeite ich noch ein. Mal sehen, ob ich das morgen noch schaffe.